source/docs/appendices/migration-guides.md

```diff
@@ -20,6 +20,16 @@ These new features may be relevant for your existing application:
 - [A quickorder module has been created to order by SKU](/docs/advanced/features/quickorder.html)
 - Front-Commerce dependencies are now regularly (and automatically) updated using [Depfu](https://depfu.com/). Patches are automatically applied if the CI is green, minor versions are manually approved. We always review changelogs provided by Depfu. **You can review updates any time [by filtering Merge Requests tagged `depfu`](https://gitlab.com/front-commerce/front-commerce/-/merge_requests?scope=all&state=merged&label_name[]=depfu).**
 
+### Storybook 6
+
+Your styleguide is now powered by Storybook 6. 
+
+This induces several changes in your custom stories.
+
+- The organization of stories is now only manageable with `/`, please set your story name like this : `section/path/of/your/story/Component`
+- As knobs are now deprecated, we highly encourages you to use controls from now on. see [the official documentation for more details](https://storybook.js.org/addons/@storybook/addon-knobs)
```

Comment:
Due to the impact, I think further references would be needed here. Examples:
- https://github.com/storybookjs/storybook/blob/next/addons/controls/README.md#how-do-i-migrate-from-addon-knobs
- https://storybook.js.org/docs/react/essentials/controls
- https://github.com/storybookjs/storybook/discussions/15060

Also a diff summarizing how to do it (from one of our updated component for instance) would make things clearer for FC.

Finally, I think we should mark it as deprecated in FC until 3.0 and document it here so that integrators can plan the migration on their side (it's not strictly needed when upgrading to 2.11)

Reply:
I did not write this doc as I don't understand myself how to do this migration and maintain everything working